.travis.yml

```diff
@@ -33,11 +33,11 @@ matrix:
       env: GROUP=examples EXTRA_CMD=--no-js
 
     # linting, package and docs building (python 3.4)
-    - python: 3.4
+    - python: 3.6
       env: GROUP=flake_docs
 
     # integration tests (python 3.4)
```

Comment:
Comments don't match. Actually I'm not sure what's the point of this repetition.